orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ServerGroupCacheForceRefreshTask.groovy

```diff
@@ -55,13 +55,15 @@ class ServerGroupCacheForceRefreshTask extends AbstractCloudProviderAwareTask im
 
   @Override
   TaskResult execute(Stage stage) {
+    
     if ((clock.millis() - stage.startTime) > autoSucceedAfterMs) {
       /*
        * If an issue arises performing a refresh, wait at least 10 minutes (the default ttl of a cache record) before
        * short-circuiting and succeeding.
        *
        * Under normal operations, this refresh task should complete sub-minute.
        */
+      log.warn("Force cache refresh issue arose. Shorting the circuit.")
```

Comment:
Could you log the time it took to short circuit as well?

Reply:
Sure.